packages/platform-sdk/test/integration-tests/process/process.test.ts

```diff
@@ -150,4 +122,34 @@ describe('integration test for process function', () => {
       })
       .catch(fn)
   })
+
+  test('should properly the `expan` args as a uri parameter', () => {
+    const client = new ClientBuilder()
+      .withProjectKey(projectKey)
+      .withClientCredentialsFlow(getAuthOptions())
+      .withHttpMiddleware(getHttpOptions())
+      .build()
+
+    const apiRoot = createApiBuilderFromCtpClient(client)
+
+    const request = apiRoot
+      .withProjectKey({ projectKey })
+      .categories()
+      .get({
+        queryArgs: {
+          limit: 2,
+          withTotal: false,
+          expand: ['parent', 'ancestors[*]'],
+        },
+      })
+      .clientRequest()
+
+    const fn = (data: any) => data
+    return client
+      .process(request, fn, {})
+      .then((response) => {
+        expect(response[0].statusCode).toEqual(200)
```

Comment:
I tried to run this test also in master and it is passing as well, so it does not show that the error is fixed or not. Please fix this IT so that it fails on master branch and not in your branch to show that the error is fixed.

Reply:
I don't know what test you ran but the issue is on master, there is nothing wrong with the test that needs fixing.

Here is the same issue in the master branch.


![ezgif-7-ea06c3391e](https://github.com/commercetools/commercetools-sdk-typescript/assets/32770340/5eb7a70a-022e-4b4f-9514-62cfa6179fb9)

Reply:
And this is a commit before the bug was introduced `d06e0c5`. 

Same code snippet ran with the commit can be seen in the attached clip and it doesn't have the said bug.

I suggest you properly test this from your end.

If you need an help with this let me know.

Thank you.

https://github.com/commercetools/commercetools-sdk-typescript/assets/32770340/cbd6d58d-22ca-43c3-95ba-8f171c899166

Reply:
Thanks for the snippets. I tried again and the test is still passing. My procedure is following:
1. I copied only the test `should properly the `expan` args as a uri parameter` to master branch.
2. I ran the command `npm test -- process.test.ts` in terminal.
3. The tests are passing and I would expect one test to fail.

I see in the snippet you don‘t do the same thing. Can you try what I did?